### PR TITLE
garbage collection for acr

### PR DESCRIPTION
**Pull request by lovew-lc**
Allows deletion of images inside ACR

### CODE REVIEW

File: templates/default/dockerImage.sh.erb [hunk 1]
```diff
@@ -46,6 +46,27 @@ elif [ "$1" == "gc" ] ; then
     docker exec registry bin/registry garbage-collect /etc/docker/registry/config.yml -m
 	  docker container restart registry
 
+elif [ "$1" == "delete-acr" ] ; then
```
Comment (maismail) [inline templates/default/dockerImage.sh.erb:49]: This should be under the else of this condition when `managed_docker_registry` is true 
https://github.com/logicalclocks/hopsworks-chef/pull/570/files#diff-944e67fb3fc477da8e732aeac8d41b01b35b98af540418d278610340363d1454R30

**Review by maismail — CHANGES_REQUESTED** [review]

File: templates/default/dockerImage.sh.erb (hunk 1 above)
Comment (lovew-lc) [reply templates/default/dockerImage.sh.erb]: ahhh I read if managed=true. fixed in last commit

Comment (maismail) [reply templates/default/dockerImage.sh.erb]: The block was correct you just need to add `<% else -%>` before it

Comment (lovew-lc) [reply templates/default/dockerImage.sh.erb:51]: Ah sorry, I am not awake today.. fixed now

Comment (maismail) [reply templates/default/dockerImage.sh.erb:51]: no worries

**Review by maismail — APPROVED** [review]

**Pull request merged**